imageroot/bin/node_exporter_handler

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2023 Nethesis S.r.l.
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+import json
+import os
+import sys
+
+import agent
+
+if len(sys.argv) != 1:
+    raise ValueError('Missing path to generate provider from.')
+
+data = {
+    'hosts': []
```

Comment:
Inside NextSecurity, instead of node-exporter-lua, we are going to use netdata: https://github.com/NethServer/nextsecurity/pull/104

Add metric path:
```suggestion
    'hosts': [],
    "labels": { "__metrics_path__": "/api/v1/allmetrics?format=prometheus&help=yes" }
```